SpecFlowProject10/Form/CalculatorForm.cs

```diff
@@ -111,17 +73,14 @@ public void EnterMode(string Mode)
             clickscientific.Click();
         }
 
-        public string GetResult()
+        public static string GetResult()
         {
-            //AppWindow appWindow = AppManager.AppGetWindow(); // Assuming this returns the Calculator's window
-            //TextBox resultTextBox = appWindow.appWindow.Get<TextBox>(SearchCriteria.ByAutomationId("150")); // Replace with the actual AutomationId of the result textbox
             Label resultTextBox = AppManager.AppGetWindow().appWindow.Get<Label>(SearchCriteria.ByAutomationId("158"));
```

Comment:
Move to class variable
Use ApplLabel wraper

Reply:
fixed